dataflow/src/main/java/org/checkerframework/dataflow/expression/JavaExpression.java

```diff
@@ -531,6 +506,50 @@ public static JavaExpression fromTree(
         return result;
     }
 
+    /**
+     * Returns the Java expression corresponding to the given variable tree {@code tree}.
+     *
+     * @param tree a variable tree
+     * @return a JavaExpression for {@code tree}
+     */
+    public static @Nullable JavaExpression fromVariableTree(VariableTree tree) {
+        return fromVariableElement(TreeUtils.typeOf(tree), TreeUtils.elementFromDeclaration(tree));
+    }
+
+    /**
+     * Returns the Java expression corresponding to the given variable element {@code ele}.
```

Comment:
Under what circumstances does it return `null`?
Could you change the `return null` clause to throw an exception?